### PR TITLE
Improve Search

### PR DESCRIPTION
**Pull request by madalinamiron**
Allow smart search to filter regarding also to city not only title.

### CODE REVIEW

File: ShoeBox/Classes/GUI/LocationScreen/ViewController/LocationsViewController.swift [hunk 1]
```diff
@@ -50,8 +50,11 @@ class LocationsViewController: UITableViewController {
             
             self.filteredLocations = self.locationsData.filter({ (location: Location) -> Bool in
                 guard let title = location.title else { return false }
+                guard let city = location.city else { return false }
                 
-                return title.lowercased().contains(searchText.lowercased())
+                let composedString = title + city
```
Comment (IonutGavris) [inline ShoeBox/Classes/GUI/LocationScreen/ViewController/LocationsViewController.swift]: You could also add the Country in the search string.

Comment (madalinamiron) [reply ShoeBox/Classes/GUI/LocationScreen/ViewController/LocationsViewController.swift]: You are so right ! I missed the country was also displayed :) Thanks !

Comment (madalinamiron) [reply ShoeBox/Classes/GUI/LocationScreen/ViewController/LocationsViewController.swift]: Done.

**Comment by IonutGavris** [comment]
Please update app version to 3.2 as iTunes refuses the build because app 3.1 was already launched.

**Comment by IonutGavris** [comment]
🎄 🎅 👍

**Pull request merged**